### DISCUSSION TITLE
支付宝支付成功后，没有升级成专业用户，请帮忙处理下，感谢

### DISCUSSION THREAD

**Discussion by liuchao233** [getlantern/lantern-forum-cn#900] [General]
支付宝支付成功后，没有升级成专业用户，也没有收到active code，请帮忙处理下。

我的账号是 15609831969@163.com

![WechatIMG46](https://github.com/user-attachments/assets/63f1f06a-3854-447a-98ca-e735367fd1be)

**Comment by MaxMKMax** [comment]
@liuchao233 您好. 好的, 我已经帮您提交问题给支付部门了. 他们会帮您处理. 谢谢您支持蓝灯!
